frontend/src/views/student/submissions/ShowSubmissionDialog.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-dialog
+    :value="dialog"
+    @input="$emit('dialog', false)"
+    @keydown.esc="$emit('dialog', false)"
+    max-width="75%"
+  >
+    <v-card>
+      <v-card-title>
+        <span class="headline">{{ submission.question.title }}</span>
+      </v-card-title>
+
+      <v-card-text class="text-left">
+        <show-question :question="submission.question" />
+      </v-card-text>
+      <v-card-title>
+        <span class="headline">{{ 'Reviews' }}</span>
+      </v-card-title>
+      <div class="text-left">
+        <show-reviews
+          class="history"
+          :key="reviewsComponentKey"
+          :submission="submission"
+        />
+      </div>
+      <div
+        class="text-left"
+        v-if="
+          submission.question.status === 'IN_REVISION' ||
+            submission.question.status === 'IN_REVIEW'
+        "
+      >
+        <v-card-title>
+          <span class="headline">{{ 'New Comment' }}</span>
+        </v-card-title>
+        <v-card-text class="text-left">
+          <v-textarea
+            rows="1"
+            v-model="comment"
+            label="Comment"
+            data-cy="Comment"
+          ></v-textarea>
+        </v-card-text>
+      </div>
+      <v-card-actions>
+        <v-spacer />
+        <v-btn color="blue darken-1" data-cy="close" @click="$emit('dialog')"
+          >close</v-btn
+        >
+        <v-btn
+          v-if="
+            submission.question.status === 'IN_REVISION' ||
```

Comment:
One way, not sure if there isn't a better one, to avoid these string is to have a methos in the submission class that returns a boolean, this way, these strings are encapsulated